Лабораторная работа 1/task 2.py

```diff
@@ -0,0 +1,4 @@
+# TODO Найдите количество книг, которое можно разместить на дискете
+one_book_bytes = 4 * 25 * 50 * 100
+volume_in_bytes = 1.44 * 1024 * 1024
```

Comment:
Верное решение!
Но внесу пару корректировок:
В вашем коде есть "магические" числа.
Задания лучше решать как задачи по физике или математике. Для каждого числа необходимо создавать свою переменную.
Например:
```suggestion
disk_mb = 1.44
pages = 100
```
И так далее.
таким образом, ваш код будет более понятен человеку, который не знает суть Вашего задания. Код будет более читаемым.